README.md

```diff
@@ -57,14 +57,14 @@ In your `ios` folder, run `pod install`. If you've just upgraded ExpoKit, you mi
 
 ##### Make a Framework Reference in your project
 
-1. Drag `Purchases.framework` and `PurchasesHybridCommon.framework` from the `RNPurchases`sub-project under the libraries section to the outer project and create a reference. 
```

Comment:
hopefully this entire process gets deleted soon, but that's a conversation on a separate PR (https://github.com/RevenueCat/react-native-purchases/pull/323) and shouldn't affect this one